lib/tasks/manage.rake

```diff
@@ -39,7 +39,9 @@ namespace :manage do
 
   desc "Move all subscribers from one subscriber list to another"
   task :move_all_subscribers, %i[from_slug to_slug] => :environment do |_t, args|
-    SubscriberListMover.new(**args).call
+    send_email = ENV["SEND_EMAIL"] || false
```

Comment:
Minor: `|| false` is redundant, because `||` only happens when the left-hand side is `false`y.